yoti-sdk-api/src/main/java/com/yoti/api/client/shareurl/DynamicScenario.java

```diff
@@ -11,7 +11,7 @@
 public interface DynamicScenario {
 
     /**
-     * The device's callback endpoint. Must be a URL relative to the Application Domain specified in your Dashboard
+     * The device's callback endpoint. Must be a URL relative to the Application Domain specified in your Hub
```

Comment:
Does this need to be "Yoti Hub"?

Reply:
Yes, I think we might as well put Yoti in to make sure we're clear. Also I'm not sure if `in Yoti Hub` is better than `in your Yoti Hub`?

```suggestion
     * The device's callback endpoint. Must be a URL relative to the Application Domain specified in Yoti Hub
```